### PR TITLE
Epic/page types

### PR DESCRIPTION
**Pull request by andij**
Improvements and changes to our Page types

### CODE REVIEW

File: docs/page-types/roadblock.md [hunk 1]
```diff
@@ -43,9 +27,9 @@ The copy must be in plain English, and if possible help the user out of the situ
 
 You should not use:
 
-- Technical jargon like ‘form post error’, ‘unspecified error’ and ‘error 0x0000000643’
-- Humourous, informal language like ‘oops’
-- Negative language
+* Technical jargon like ‘form post error’, ‘unspecified error’ and ‘error 0x0000000643’
```
Comment (dbenmore) [inline docs/page-types/roadblock.md:30]: This error code is a cut and paste from gov.uk. Let's pt something original :) "0x0000000643"

File: includes/storybook-page.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (dbenmore) [inline includes/storybook-page.js:1]: note for future: use `git mv` to rename files to preserve their history

File: src/css/custom.css [hunk 3]
```diff
@@ -46,13 +46,21 @@
...
   border-radius: var(--ifm-global-radius);
   z-index: calc(var(--ifm-z-index-fixed) - 1);
 }
 
+.storybook-embed.page iframe {
+  width: auto;
+  border-top-width: 64px;
+  border-bottom-width: 64px;
+  min-width: 399px;
+  min-height: 795px;
```
Comment (dbenmore) [inline src/css/custom.css:61]: could we use one of our viewports here? or if they are too small/big a more common size such as the iPhone 6/7/8 (375 * 667)

File: docs/page-types/guide.md [hunk 4]
```diff
@@ -3,32 +3,32 @@ title: Guide
...
-https://www.britishgas.co.uk/energy/thinking-of-going-electric.html
-
-## Principles to follow
+<StorybookPage story="examples-page-types--guide"></StorybookPage>
 
-Clean and fresh
+## Content guidance
 
-## Tone of voice
+[TODO]
```
Comment (dbenmore) [inline docs/page-types/guide.md:25]: I'm not so keen on todo in documentation as it doesn't instil confidence - they often get forgotten or not get worked on for a long time.. Rather than todo could we either link to an issue, ask them to contact us or remove it entirely

**Comment by dbenmore** [comment]
Do you think it's worth adding what components were used and why to each of these pages? this could be split into a new issue if you agree

File: src/css/custom.css (hunk 3 above)
Comment (andij) [reply src/css/custom.css:61]: The numbers are odd due to the border on the iframe. The page within the phone frame is 375 * 667.

image

File: includes/storybook-page.js (hunk 2 above)
Comment (andij) [reply includes/storybook-page.js:1]: Noted, thanks. That's actually a new one for me, I can't think that I've ever used `git mv`.

File: docs/page-types/guide.md (hunk 4 above)
Comment (andij) [reply docs/page-types/guide.md:25]: Yeah, I agree. I'll tidy these sections up.

**Comment by andij** [comment]
> Do you think it's worth adding what components were used and why to each of these pages? this could be split into a new issue if you agree

Yeah, totally agree. It's in scope for the broader piece of work on these page types. It's one of the points in the Assessment section within the tickets we raised for each page type.

The scope for each page type is the same as this: https://github.com/ConnectedHomes/nucleus/issues/2135

File: docs/page-types/roadblock.md (hunk 1 above)
Comment (andij) [reply docs/page-types/roadblock.md]: I've changed this to be more a more generic error message.

**Review by dbenmore — APPROVED** [review]
